controller_manager:
  ros__parameters:
    update_rate: 100.0

    joint_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController
      joints:
        - shoulder_pan_joint
        - shoulder_lift_joint
        - elbow_joint
        - wrist_1_joint
        - wrist_2_joint
        - wrist_3_joint
      state_publish_rate: 50.0
      action_ns: follow_joint_trajectory  # This should match the action name used in your ActionClient

      gains:
        shoulder_pan_joint: {p: 100.0, d: 1.0, i: 0.0}
        shoulder_lift_joint: {p: 100.0, d: 1.0, i: 0.0}
        elbow_joint: {p: 100.0, d: 1.0, i: 0.0}
        wrist_1_joint: {p: 100.0, d: 1.0, i: 0.0}
        wrist_2_joint: {p: 100.0, d: 1.0, i: 0.0}
        wrist_3_joint: {p: 100.0, d: 1.0, i: 0.0}
